### PR TITLE
[backend] created and modified attributes not available for observables (#6779)

### PR DESCRIPTION
**Pull request by Archidoit**
issue: https://github.com/OpenCTI-Platform/opencti/issues/6779

### CODE REVIEW

File: opencti-platform/opencti-graphql/src/schema/general.js [hunk 1]
```diff
@@ -5,7 +5,6 @@ import { ENTITY_TYPE_CONTAINER_CASE } from '../modules/case/case-types';
 export const STIX_TYPE_RELATION = 'relationship';
 export const STIX_TYPE_SIGHTING = 'sighting';
 
-export const KNOWLEDGE_ORGANIZATION_RESTRICT = 'KNORGARESTRICT';
```
Comment (Kedae) [inline opencti-platform/opencti-graphql/src/schema/general.js:8]: ?

Comment (Archidoit) [reply opencti-platform/opencti-graphql/src/schema/general.js:8]: this constant was unused

Comment (Archidoit) [reply opencti-platform/opencti-graphql/src/schema/general.js:8]: another 'KNOWLEDGE_ORGANIZATION_RESTRICT' (with different value) is defined in access.ts

Comment (Kedae) [reply opencti-platform/opencti-graphql/src/schema/general.js:8]: Oh yes, missed that 🥲

File: opencti-platform/opencti-graphql/src/modules/attributes/stixObject-registrationAttributes.ts [hunk 2]
```diff
@@ -1,11 +1,9 @@
-import { type AttributeDefinition, created, files, modified, xOpenctiStixIds } from '../../schema/attribute-definition';
+import { type AttributeDefinition, files, xOpenctiStixIds } from '../../schema/attribute-definition';
 import { schemaAttributesDefinition } from '../../schema/schema-attributes';
 import { ABSTRACT_STIX_OBJECT } from '../../schema/general';
 
 const stixObjectAttributes: Array<AttributeDefinition> = [
   xOpenctiStixIds,
-  created,
-  modified,
```
Comment (lndrtrbn) [inline opencti-platform/opencti-graphql/src/modules/attributes/stixObject-registrationAttributes.ts:8]: For my own curiosity, why do we remove those fields from `stixObjects`?

Comment (Archidoit) [reply opencti-platform/opencti-graphql/src/modules/attributes/stixObject-registrationAttributes.ts:8]: created and modified are only field existing for stix core objects and stix meta objects, not for observables (see STIX 2 documentation).
So I removed them from stixObjects , to only put them for stix core objects and stix meta objects.

Comment (labo-flg) [reply opencti-platform/opencti-graphql/src/modules/attributes/stixObject-registrationAttributes.ts:8]: they're moved in stixDomainObjects apparently.
But that's not the model definition documented... so I'm curious too :)

**Review by labo-flg — APPROVED** [review]

**Review by lndrtrbn — APPROVED** [review]

**Pull request merged**